cmake/thirdparty/get_legion.cmake

```diff
@@ -195,19 +206,19 @@ function(find_or_configure_legion)
                                  "Legion_USE_CUDA ${Legion_USE_CUDA}"
                                  "Legion_NETWORKS ${Legion_NETWORKS}"
                                  "Legion_USE_OpenMP ${Legion_USE_OpenMP}"
-                                 "Legion_USE_Python ${Legion_USE_Python}"
+                                 "Legion_USE_Python ON"
                                  "Legion_BOUNDS_CHECKS ${Legion_BOUNDS_CHECKS}"
+                                 "Legion_BUILD_JUPYTER ON"
+                                 "Legion_EMBED_GASNet_CONFIGURE_ARGS --with-ibv-max-hcas=8"
     )
   endif()
 
```

Comment:
We should probably validate that whatever Legion is found sets "Legion_USE_PYTHON"

```
macro(_check_required_legion_option option)
if (NOT ${option})
  message(FATAL_EROR "External Legion was not build with ${option}=ON")
endif()
endmacro()

_check_required_legion_option(Legion_USE_PYTHON)
_check_required_legion_option(Legion_REDOP_HALF)

```

Reply:
I am doing that around line 75 in the same file.

Legion doesn't actually export `Legion_REDOP_*` it seems, at least according to what's in https://github.com/StanfordLegion/legion/blob/stable/cmake/LegionConfigCommon.cmake.in. I realized we can actually check that in `legate_defines.h`, so the only major configuration option that's missing is `Legion_BUILD_BINDINGS`.

Reply:
whoops, I can't read